src/schemas/functions/post.ts

```diff
@@ -32,23 +32,38 @@ export async function addUser(user: CreateUser["input"]): Promise<string> {
 //User Login
 export async function login(userLogin: LoginUser["input"]) {
   const userInfo = await checkEmail(userLogin.emailAddress);
-  if (userInfo) {
-    if (await bycrypt.compare(userLogin.password, userInfo.password)) {
-      return userInfo.token;
-    } else {
-      throw new UserInputError("Invalid credentials.");
-    }
+  if (
+    userInfo &&
+    (await bycrypt.compare(userLogin.password, userInfo.password))
+  ) {
+    return issueToken(userInfo.id, userInfo.emailAddress);
   } else {
     throw new UserInputError("Invalid credentials.");
   }
 }
 
 //User
 async function getUserInfo(user: CreateUser["input"]) {
-  const id = uuidv4().replaceAll("-", "").concat("account");
+  const id = getID("account");
   const hashedPassword = await bycrypt.hash(user.password, 10);
   const email = user.emailAddress;
   const token = issueToken(id, email);
 
   return { id, email, hashedPassword, token };
 }
+
+/*************************************************************************** */
+// PRODUCTS
+
+//Add Product
+export async function addProduct(productInfo: any, userInfo: any) {
```

Comment:
Avoid using `any` type as much as possible and use specific types, especially for your function parameters.
In here, instead of `userInfo`, you can just use `userId` or `ownerId`.